channels/mx.m3u

```diff
@@ -3,29 +3,29 @@
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtit-tdt2.m3u8
 #EXTINF:-1 tvg-id="ADN40.mx" tvg-name="ADN 40" tvg-country="MX" tvg-language="" tvg-logo="" group-title="",ADN 40
 http://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master01.m3u8
-#EXTINF:-1 tvg-id="XHTVMTDT.mx" tvg-name="ADN 40 (XHTVM-TDT)" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (XHTVM-TDT) (480p)
+#EXTINF:-1 tvg-id="XHTVMTDT.mx" tvg-name="ADN 40 (XHTVM-TDT)" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="Local",ADN 40 (XHTVM-TDT) (480p)
```

Comment:
Nothing tells me that this channel is broadcasted locally. It is indeed using the San Diego transmitter, but is also being transmitted from all around Mexico.

It is a News channel.